cmd/dt/relocate/relocate.go

```diff
@@ -42,4 +44,8 @@ func NewCmd(cfg *config.Config) *cobra.Command {
 			return nil
 		},
 	}
+
+	cmd.PersistentFlags().StringSliceVar(&valuesFiles, "values", []string{"values.yaml"}, "values files to relocate images (can specify multiple)")
```

Comment:
I guess is a personal taste but i prefer setting the default value into the variable and use the variable here:

```
valuesFiles := []string{"values.yaml"}
....
cmd.PersistentFlags().StringSliceVar(&valuesFiles, "values", valuesFiles, "values files to relocate images (can specify multiple)")
```